src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -88,24 +88,24 @@ public SuccessfulTransformationResponse handle(ExceptionRecord exceptionRecord)
         CaseResponse caseTransformationResponse = caseTransformer.transformExceptionRecord(exceptionRecord, false);
 
         if (caseTransformationResponse.getErrors() != null && caseTransformationResponse.getErrors().size() > 0) {
-            log.info("Errors found while transforming exception record id {}", exceptionRecordId);
+            log.info("Errors found while transforming exception record id {} - {}", exceptionRecordId, stringJoin(caseTransformationResponse.getErrors()));
             throw new InvalidExceptionRecordException(caseTransformationResponse.getErrors());
         }
 
         if (BooleanUtils.isTrue(exceptionRecord.getIsAutomatedProcess()) && !CollectionUtils.isEmpty(caseTransformationResponse.getWarnings())) {
-            log.info("Warning found while transforming exception record id {}", exceptionRecordId);
+            log.info("Warnings found while transforming exception record id {} - {}", exceptionRecordId, stringJoin(caseTransformationResponse.getWarnings()));
```

Comment:
Here we might be logging the case sensitive information into logs. e.g invalid person name, address etc

Reply:
removed, since we needed to change validation message only